buildlogger/basic_sender.go

```diff
@@ -294,10 +298,14 @@ func (b *buildlogger) Send(m message.Composer) {
 		if line == "" {
 			continue
 		}
+		data := strings.TrimRightFunc(line, unicode.IsSpace)
+		if b.opts.Prefix != "" {
+			data = fmt.Sprintf("[%s] %s", b.opts.Prefix, data)
```

Comment:
would it be useful to have users specify a format string instead of a prefix? It would be more flexible.

Reply:
What do you mean by a format string? There is already a format option in the senders, this is a specific request for resmoke prefixes that are running in jasper processes.

Reply:
I was thinking of an option like FormatString in
```
data = fmt.Sprintf(b.opts.FormatString, data)
```
so a user could give a format string `"[my prefix] %s"` and it would be equivalent, or they could give a suffix `"%s [my suffix]"` or they could demarcate a prefix with `"{my prefix} %s"`, etc.

Reply:
Oh I don't think I would want to do that. The current users of this will be in python, converting string formatting would not be pleasant lol. For go users, they can set the `send.Sender` formatter if they want something like that.